content/applications/sales/crm/performance/expected_revenue_report.rst

```diff
@@ -0,0 +1,142 @@
+
+Click :guilabel:`New Rule`. Click in the new rule's first field, then type `Salesperson` in the
+search bar, or scroll through the list to select it. Click in the rule's second field and select
+:guilabel:`is set` from the drop-down menu. This will exclude any results without an assigned
+salesperson.
+
+.. image:: expected_revenue_report/custom-filters.png
+   :align: center
+   :alt: Add Custom Filters pop-up window with custom filters configured for expected revenue
+         report.
+
+.. important::
+   At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any`
+   or :guilabel:`all` of the rules. In order to properly run the report, only records that match
+   **all** of the following filters should be included. Before adding the filters, make sure
+   :guilabel:`all` is selected in this field.
+
+   .. image:: expected_revenue_report/match-all-filters.png
+      :align: center
+      :alt: Emphasis on the match all filters option on the Add Custom Filter pop-up window.
+
+At the bottom of the :guilabel:`Add Custom Filter` form, click :guilabel:`Add`.
+
+Save as a favorite
+------------------
+
+To save this report as a favorite, click on the arrow to the right of the search bar, then click
+:guilabel:`Save current search`. Enter a name for this report, then click :guilabel:`Save`.
+
+.. tip::
+   Before saving, additional selections can be made from this drop-down menu. For example, to view
+   expected revenue by individual salesperson, selected :guilabel:`Salesperson` under the
+   :guilabel:`Group By` heading.
+
+View options
+============
+
+The expected revenue report benefits from utilizing multiple views. The default graph view can be
+used to identify which salespeople are expected to bring in the most revenue, while the list view
+and pivot view provide more detail on specific deals.
+
+.. tabs::
+
+   .. tab:: Graph view
+
+      The *graph view* is used to visualize data, and is beneficial in identifying patters and
+      trends. *Bar charts* are used to show the distribution of data across several categories or
+      among several salespeople. *Line charts* are useful to show changing trends over a period of
+      time. *Pie charts* are useful to show the distribution or comparison of data among a small
+      number of categories or salespeople, specifically how they form the meaningful part of a whole
+      picture.
+
+      The default view for the expected revenue report is the bar chart, stacked. To change to a
+      different graph view, click one of the icons at the top-left of the report. While both the
+      line chart bar chart are available in stacked view, the pie chart is not.
+
+      .. figure:: expected_revenue_report/graph-view-icons.png
+         :align: center
+         :alt: Close up view of the graph icons on the Pipeline analysis report in the CRM app.
+
+         Graph view icons in order: bar chart, line chart, pie chart, stacked.
```

Comment:
do you think it would be beneficial to add GUIs or ** here? Or do you think it's good as-is? Curious to see what you think.

Reply:
I don't think that the guis or italics are necessary in the caption since it is just annotating the figure, not acting the same as normal text